exp/views/responses.py

```diff
@@ -39,773 +50,626 @@
+            "col [some or attending college], assoc [2-year college degree], bach [4-year college degree], "
+            "grad [some or attending graduate or professional school], prof [graduate or professional degree], "
+            "na [not applicable - no spouse or partner]"
+        ),
+        extractor=lambda resp: resp["demographic_snapshot__spouse_education_level"],
+        name="Parent education level",
+    ),
+    ResponseDataColumn(
+        id="demographic__annual_income",
+        description=(
+            "Parent's response to question 'What is your approximate family yearly income (in US dollars)?'; "
+            "options are 0, 5000, 10000, 15000, 20000-19000 in increments of 10000, >200000, or na [prefer not "
+            "to answer]"
+        ),
+        extractor=lambda resp: resp["demographic_snapshot__annual_income"],
+        name="Annual income",
+    ),
+    ResponseDataColumn(
+        id="demographic__number_of_books",
+        description="Parent's response to question 'About how many children's books are there in your home?'; integer",
+        extractor=lambda resp: resp["demographic_snapshot__number_of_books"],
+        name="Number of books",
+    ),
+    ResponseDataColumn(
+        id="demographic__additional_comments",
+        description="Parent's freeform response to question 'Anything else you'd like us to know?'",
+        extractor=lambda resp: resp["demographic_snapshot__additional_comments"],
+        name="Additional comments",
+    ),
+    ResponseDataColumn(
+        id="demographic__country",
+        description="Parent's response to question 'What country do you live in?'; 2-letter country code",
+        extractor=lambda resp: resp["demographic_snapshot__country"],
+        name="Country code",
+    ),
+    ResponseDataColumn(
+        id="demographic__state",
+        description=(
+            "Parent's response to question 'What state do you live in?' if country is US; 2-letter state "
+            "abbreviation"
+        ),
+        extractor=lambda resp: resp["demographic_snapshot__state"],
+        name="US State",
+    ),
+    ResponseDataColumn(
+        id="demographic__density",
+        description=(
+            "Parent's response to question 'How would you describe the area where you live?'; options are "
+            "urban, suburban, rural"
+        ),
+        extractor=lambda resp: resp["demographic_snapshot__density"],
+        name="Density",
+    ),
+    ResponseDataColumn(
+        id="demographic__lookit_referrer",
+        description="Parent's freeform response to question 'How did you hear about Lookit?'",
+        extractor=lambda resp: resp["demographic_snapshot__lookit_referrer"],
+        name="How you heard about Lookit",
+    ),
+]
 
-    headers_ordered = [name for (name, val, desc) in all_row_data][0:-2]
+# Which headers from the response data summary should go in the child data downloads
+child_csv_headers = [
```

Comment:
Capitalize top level "global constants" `CHILD_CSV_HEADERS` and `IDENTIFIABLE_DATA_HEADERS`

Reply:
thanks, done!